### PR TITLE
SupportIPVSProxyMode is GA

### PR DESCRIPTION
**Pull request by Lion-Wei**
Update feature gate reference for 1.11
https://github.com/kubernetes/kubernetes/pull/58442

### CODE REVIEW

**Comment by Lion-Wei** [comment]
/cc @m1093782566 
/assign @chenopis 
Please take a look, thanks.

**Comment by tengqm** [comment]
Please do not touch the autogenerated files.
@Bradamant3 I have seen quite a few PRs attempting to manually change the generated files. Maybe our last move to the relocate these files out of a 'generated' directory is leading to such a situation. What do you think?

**Comment by Lion-Wei** [comment]
@tengqm Sure, sorry didn't notice. Fixed. : )

File: content/en/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -382,7 +382,7 @@ kubelet
       <td colspan="2">--feature-gates mapStringBool</td>
```
Comment (mistyhacks) [inline content/en/docs/reference/command-line-tools-reference/kubelet.md]: Revert changes to this file, as it is generated. These docs will be updated as part of the 1.11 docs release process.

Comment (m1093782566) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: @Lion-Wei Can you fix it?

Comment (Lion-Wei) [reply content/en/docs/reference/command-line-tools-reference/kubelet.md]: Sure,

**Comment by Lion-Wei** [comment]
@mistyhacks Fixed, thanks.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -85,8 +85,9 @@ different Kubernetes components.
 | `StorageObjectInUseProtection` | `true` | GA | 1.11 | |
 | `StreamingProxyRedirects` | `true` | Beta | 1.5 | |
 | `SupportIPVSProxyMode` | `false` | Alpha | 1.8 | 1.8 |
-| `SupportIPVSProxyMode` | `false` | Beta | 1.9 | 1.9 |
-| `SupportIPVSProxyMode` | `true` | Beta | 1.10 | |
+| `SupportIPVSProxyMode` | `false` | Beta | 1.9 | |
```
[the remark this answers is not in the document]
Comment (Lion-Wei) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: I think this column means the last version that we can use `SupportIPVSProxyMode` feature gate, and that should be 1.10, so I deleted `1.9`.

[the remark this answers is not in the document]
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: The correct way to interpret this *line* is "the feature foo was in Beta and set to false starting 1.9, and that state ends at 1.9". 
Then we have the next line, "the feature was still Beta, but default to true starting 1.10, and that state ends at 1.10".
Then the next line "the feature is GA starting 1.11 and default to True, and that state persists ..."

You are *not* supposed to remove the last column on line 88.

[the remark this answers is not in the document]
Comment (Lion-Wei) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Right, I see, will change soon. Thanks.

**Comment by m1093782566** [comment]
LGTM in technical view.

**Comment by tengqm** [comment]
@mistyhacks Approved and then realized that this should be handed over to you because it is 1.11.

**Comment by mistyhacks** [comment]
I rebased to resolve conflicts

**Comment by m1093782566** [comment]
Thanks all!

Let's merge this PR.

/lgtm

**Comment by tengqm** [comment]
@zacharysarah Thanks.

**Comment by zacharysarah** [comment]
@tenqm For what to do differently next time: if you realize that you've mistakenly approved a PR, `/approve cancel` as soon as possible. In the case of this PR, the `/approve` remained after you caught the mistake, allowing another user to `/lgtm` and `/hold cancel` to merge.

**Comment by tengqm** [comment]
@zacharysarah got it.

**Pull request merged**